### PR TITLE
[IMP] (odoo): rename wording of canceled to cancelled

### PR DESCRIPTION
**Pull request by kusi-web**
In this PR by making this change, Odoo will have consistent spelling in line with US English standards, creating a more unified and polished user experience for our US-based users. 
- Identify all occurrences of the word "Cancelled" in the Odoo codebase.
- Replace each occurrence with the correct spelling, "Canceled".
- Ensure that the change only affects the spelling of the word and does not alter any other part of the code.

task-3387055

### CODE REVIEW

**Comment by tivisse** [comment]
Can you please squash all your commits and make the runbot green ? (Including upgrade builds).

Thanks :)

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: addons/payment_adyen/const.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
         'RedirectShopper'
     ),
     'done': ('Authorised',),
-    'cancel': ('Cancelled',),
+    'cancel': ('Canceled',),
```
[the remark this answers is not in the document]
Comment (ppr-odoo) [reply addons/payment_adyen/const.py]: @AntoineVDV Okay Thanks

**Review by AntoineVDV — APPROVED** [review]

File: addons/purchase/views/portal_templates.xml [hunk 2]
```diff
@@ -113,7 +113,7 @@
                               <i class="fa fa-fw fa-file-text" role="img" aria-label="Waiting for Bill" title="Waiting for Bill"></i><span class="d-none d-md-inline"> Waiting for Bill</span>
                           </span>
                           <span t-if="order.state == 'cancel'" class="badge rounded-pill text-bg-secondary">
-                              <i class="fa fa-fw fa-remove" role="img" aria-label="Cancelled" title="Cancelled"></i><span class="d-none d-md-inline"> Cancelled</span>
+                              <i class="fa fa-fw fa-remove" role="img" aria-label="Canceled" title="Canceled"></i><span class="d-none d-md-inline"> Canceled</span>
```
Comment (oomsveta) [inline addons/purchase/views/portal_templates.xml:116]:
```suggestion
                              <i class="fa fa-fw fa-remove" aria-hidden="true"></i><span class="d-none d-md-inline"> Canceled</span>
```

File: addons/stock_delivery/models/stock_picking.py [hunk 3]
```diff
@@ -256,7 +256,7 @@ def open_website_url(self):
     def cancel_shipment(self):
         for picking in self:
             picking.carrier_id.cancel_shipment(self)
-            msg = "Shipment %s cancelled" % picking.carrier_tracking_ref
+            msg = "Shipment %s canceled" % picking.carrier_tracking_ref
```
Comment (oomsveta) [inline addons/stock_delivery/models/stock_picking.py:259]: Note: This message does not appear to be translatable

File: addons/web/static/lib/owl/owl.js [hunk 4]
```diff
@@ -1641,7 +1641,7 @@
         return fiber;
     }
     function throwOnRender() {
-        throw new OwlError("Attempted to render cancelled fiber");
+        throw new OwlError("Attempted to render canceled fiber");
```
Comment (oomsveta) [inline addons/web/static/lib/owl/owl.js:1644]: Aren't you supposed to make a PR to the [Owl repository](https://github.com/odoo/owl)?

**Review by oomsveta — COMMENTED** [review]
Hello, great initiative! 😄

**Comment by oomsveta** [comment]
Harmonizing the spelling used in Odoo is a good thing, but on second thought, I wonder if we're doing it right. Every American I've asked has told me they've never heard of "canceled" and that it even looks wrong. Even though it's the spelling recommended by some dictionaries, it's still struggling to gain acceptance.

I think "canceled" with a single L makes more sense, but I also believe that the question here is which spelling will be best received, and when it comes to spelling, my suspicion is that people are usually more attached to familiarity than rationality. Perhaps we should keep using " cancelled" with two L's if we can confirm that's the one Americans are most familiar with. Ultimately, I think it's more of a marketing decision

**Comment by oomsveta** [comment]
Regarding my previous comment: I heard that the US marketing team validated "canceled" with a single L.
Don't forget to change "cancellation", "cancelling", and "canceller" as well

**Comment by ppr-odoo** [comment]
@oomsveta Thanks for the suggestion. I have made changes. Please look at.

**Comment by sofiagvaladze** [comment]
Hello Hello, @kusi-odoo, @ppr-odoo  with a little bit of discussion with @Geoffrey-GMF and @ticodoo,

It seems that opposite is a better idea actually :innocent: 

After all, `canceled` is not okey by many connectors and on top of that, even though dictionary says otherwise, seems like Americans don't use it. :woman_shrugging: 

All of this to say, thanks for your work but we should be doing opposite. Converting `canceled` to `cancelled` everywhere.

Sorry for the inconvenience and let me know what you guys think? 

cc: @oomsveta

File: addons/payment_stripe/const.py [hunk 5]
```diff
@@ -42,7 +42,7 @@
     'pending': ('processing', 'pending'),
     'authorized': ('requires_capture',),
     'done': ('succeeded',),
-    'cancel': ('canceled',),
```
Comment (AntoineVDV) [inline addons/payment_stripe/const.py:45]: `canceled` is the correct one for Stripe

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Review by AntoineVDV — APPROVED** [review]

**Comment by sofiagvaladze** [comment]
robodoo r+

**Review by tde-banana-odoo — CHANGES_REQUESTED** [review]
Please don't change keys in mail models. Those are heavily used models, and changing keys has a cost in migration as well as bug fix it creates. Change labels if you want, but don't change models.

**Comment by tde-banana-odoo** [comment]
@robodoo r-

**Comment by sofiagvaladze** [comment]
robodoo r-

**Comment by sofiagvaladze** [comment]
@robodoo r+

**Pull request closed without merging**